lntest/itest/lnd_test.go

```diff
@@ -15090,10 +15100,24 @@ func testExternalFundingChanPoint(net *lntest.NetworkHarness, t *harnessTest) {
 		t.Fatalf("unable to make payments between Carol and Dave")
 	}
 
-	// To conclude, we'll close the newly created channel between Carol and
-	// Dave.
-	closeChans()
+	// Now that the channels are open, and we've confirmed that they're
+	// operational, we'll now ensure that the channels are frozen as
+	// intended (if requested).
+	//
+	// First, we'll try to close the channel as Carol, the initiator. This
+	// should fail as a frozen channel only allows the responder to
+	// initiate a channel close.
```

Comment:
It would also be nice to test the responder preventing a close initiated by the other party, but that seems a bit more involved.

Reply:
Yeah, we'll need to do some refactoring for the `chanCloser` so we can test that on the unit level. Conner has an overhaul of the logic planned, so perhaps we can add this test at that time.